src/strings-in-block/class-html.php

```diff
@@ -211,12 +211,28 @@ private function getTranslation( $text, $lang, \WP_Block_Parser_Block $block, ar
 		if ( $translationFromPageBuilder === $text ) {
 			$string_id = $this->get_string_id( $block->blockName, $text );
 			if ( Obj::path( [ $string_id, $lang, 'status' ], $string_translations ) == ICL_TM_COMPLETE ) {
-				return $string_translations[ $string_id ][ $lang ]['value'];
+				return self::preserveNewLines( $text, $string_translations[ $string_id ][ $lang ]['value'] );
 			} else {
 				return null;
 			}
 		} else {
 			return $translationFromPageBuilder;
 		}
 	}
+
+	private static function preserveNewLines( $original, $translation ) {
```

Comment:
This adds leading and trailing newlines to the translation if they are missing and in the original. It's required so the embeds work correctly. To me it seems safe to do it for all blocks and not just embeds.

@strategio What do you think?

Reply:
I agree with you @brucepearson, I believe this can apply to all blocks.